python-package/lightgbm/dask.py

```diff
@@ -492,7 +545,14 @@ def to_local(self):
 class DaskLGBMRegressor(LGBMRegressor, _DaskLGBMModel):
     """Distributed version of lightgbm.LGBMRegressor."""
 
-    def fit(self, X, y, sample_weight=None, client=None, **kwargs):
+    def fit(
+        self,
+        X: _DaskCollection,
```

Comment:
Same about inability to pass `dask Series` as above.

Reply:
fixed in https://github.com/microsoft/LightGBM/pull/3866/commits/8b19e04b05f83a38f5213c1103040f49d0a20988